tests/e2e/run-compliance-e2e.sh

```diff
@@ -35,6 +35,11 @@ test_compliance_e2e() {
     install_the_compliance_operator
     deploy_stackrox
 
+    # This is required until the deployment tooling (helm or the ACS operator)
+    # updates the stackrox service account permissions so that it can make
+    # changes to Compliance Operator resources. This is being tracked in ROX-18124.
```

Comment:
Looks like this is already being addressed in https://github.com/stackrox/stackrox/commit/ecb7927bc816d2ce6f1e86113b4a271d19f6516a